spec/support/pipelines/requestor_controller.rb

```diff
@@ -1,171 +0,0 @@
-require 'rails_helper'
```

Comment:
Where there any other `Pipeline::Requestor: ...` spec files to delete?

Reply:
Yes. There are some left. There is still a bit of work to do but I decided to timebox the refactoring for now. I am also doing the Rails 7 upgrade as part of DPL-274 so lots of improvements happening.